webview-ui/src/manualTest/main.tsx

```diff
@@ -46,6 +46,7 @@ const contentTestScenarios: Record<ContentId, Scenario[]> = {
     draftDeployment: getDraftDeploymentScenarios(),
     draftDockerfile: getDraftDockerfileScenarios(),
     draftWorkflow: getDraftWorkflowScenarios(),
+    draftValidate: [],
```

Comment:
💡 Thanks for taking care of other 2 comments, just one last: Might be jsut a future work comment here, as this is empty for the sake of cover (I think) - I. have re-approved this.

Reply:
Yes exactly. In the future when the webview gets more complex we'll add some test cases